### PR TITLE
Add uphold scripts

### PR DESCRIPTION
**Pull request by cristianooliveira**
## Description

Add uphold scripts.
Add yarn.lock.

## Motivation

The initial motivation for this PR was to remove security vulnerabilities related with old packages.

### CODE REVIEW

**Comment by waldyrious** [comment]
Please split the changes into atomic commits.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
 
-## [v1.0.1](https://github.com/seegno/range-specifier-parser/tree/v1.0.1) (2016-05-18)
-[Full Changelog](https://github.com/seegno/range-specifier-parser/compare/v1.0.0...v1.0.1)
+## [1.0.2](https://github.com/uphold/range-specifier-parser/releases/tag/v1.0.2) (2019-11-11)
+- Add MIT license [\#6](https://github.com/uphold/range-specifier-parser/pull/6) ([cristianooliveira](https://github.com/cristianooliveira))
+- Improve readme and add release instructions [\#5](https://github.com/uphold/range-specifier-parser/pull/5) ([nunofgs](https://github.com/nunofgs))
+- Fix parsing of falsy range [\#4](https://github.com/uphold/range-specifier-parser/pull/4) ([nunofgs](https://github.com/nunofgs))
 
-**Merged pull requests:**
+## [v1.0.0](https://github.com/uphold/range-specifier-parser/releases/tag/v1.0.0) (2016-05-18)
```
Comment (cristianooliveira) [inline CHANGELOG.md:8]: This seems to be incorrect because of the tags and pull requests.

Comment (rplopes) [reply CHANGELOG.md:8]: Indeed. The releases for v1.0.1 and v1.0.2 have never been released on GitHub, so this generator isn't counting with them.

File: package.json [hunk 2]
```diff
@@ -2,36 +2,26 @@
...
   "description": "Range Specifier Parser",
-  "homepage": "https://github.com/seegno/range-specifier-parser",
+  "homepage": "https://github.com/uphold/range-specifier-parser",
+  "repository": "git@github.com:uphold/range-specifier-parser",
   "license": "MIT",
-  "author": "Seegno",
-  "main": "dist/src",
-  "repository": "git@github.com:seegno/range-specifier-parser",
+  "author": "Uphold",
+  "main": "src/index.js",
```
Comment (rplopes) [inline package.json:9]: This change seems to belong to the separate PR of removing babel.

File: package.json [hunk 3]
```diff
@@ -2,36 +2,26 @@
...
   "scripts": {
-    "changelog": "github_changelog_generator --no-issues --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,4ba' -e 'P;D' CHANGELOG.md",
-    "lint": "eslint src test && jscs src test",
-    "test": "NODE_ENV=test mocha $npm_package_options_mocha",
-    "version": "npm run changelog --future-release=$npm_package_version && git add -A CHANGELOG.md"
+    "changelog": "uphold-scripts changelog $npm_package_version",
```
Comment (rplopes) [inline package.json]:
```suggestion
    "changelog": "uphold-scripts changelog v$npm_package_version",
```

Since this project seems to prefer to use the `v` prefix for the release names.

File: src/index.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ module.exports = range => {
   // Pick `first-byte-pos` and `last-byte-pos` from `range-set`.
   const first = parseInt(rangeSet[0], 10);
   const last = isFinite(rangeSet[1]) ? parseInt(rangeSet[1], 10) : rangeSet[1];
+  // eslint-disable-next-line no-useless-escape
```
Comment (rplopes) [inline src/index.js]: Fix linting error instead?

**Review by rplopes — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (waldyrious) [reply package.json:9]: This change has been moved to #11.

File: CHANGELOG.md (hunk 1 above)
Comment (cristianooliveira) [reply CHANGELOG.md:8]: Maybe we should do that releases on GitHub?

**Review by waldyrious — CHANGES_REQUESTED** [review]
CHANGELOG fix needed before this can be merged (as mentioned [above](https://github.com/uphold/range-specifier-parser/pull/8#discussion_r345248585)).

**Comment by cristianooliveira** [comment]
> CHANGELOG fix needed before this can be merged (as mentioned [above](https://github.com/uphold/range-specifier-parser/pull/8#discussion_r345248585)).

@waldyrious I was looking at this issue and found the problem was related with the merged date of the PR and date of creation of the release.

What we can do here is update one of this dates to fit in the other. 

Is it worth to do it? What do you think?

**Comment by waldyrious** [comment]
Digging into the git graph a bit, it looks like the issue is that the `v0.1.0` tag was created *after* the merge of PRs #2 and #3, but pointing to a commit that is located *before* them in the git history.

- PR #2 was merged at `2016-05-18 15:24`
- PR #3 was merged at `2016-05-18 15:37`
- release [v0.1.0](https://github.com/uphold/range-specifier-parser/releases/tag/v1.0.0) was created at `2016-05-18 15:57`

The changelog generator is therefore assuming that those PRs are part of that release, because it considers the date rather than the git graph topology to calculate which PRs go into which releases. That's clearly a bug in the changelog generator.

Until that bug is fixed, this changelog will not be correctly generated, so as a workaround, I'd suggest simply removing the [v0.1.0 release](https://github.com/uphold/range-specifier-parser/releases/tag/v1.0.0) (leaving the tag), hoping that the generator will then place #2 and #3 in the correct release, `v1.0.0`. That should be fine since `v0.1.0` of this repo [does not seem to be used by any other project](https://github.com/search?q=org%3Auphold+range-specifier-parser&type=Code).

**Comment by cristianooliveira** [comment]
Closed due to inactivity.
